vowpalwabbit/cb_sample.cc

```diff
@@ -8,82 +8,91 @@ using namespace LEARNER;
 using namespace VW;
 using namespace VW::config;
 
+namespace VW
+{
 struct cb_sample_data
```

Comment:
Some explanation of what this reduction does will be useful.

Reply:
Sounds good, I'll add a doc comment